### PR TITLE
Bump pykronos dependency

### PR DESCRIPTION
**Pull request by derek-schultz**

### CODE REVIEW

File: kronos/requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ gevent==1.0.1
 gipc==0.4.0
 greenlet==0.4.2
 lz4==0.7.0
-pykronos==0.6.0
+pykronos==0.6.2
```
Comment (marcua) [inline kronos/requirements.txt]: does Kronos really depend on pykronos?  Is it for tests?

**Comment by marcua** [comment]
quick q then we're good

File: kronos/requirements.txt (hunk 1 above)
Comment (derek-schultz) [inline kronos/requirements.txt]:
```
kronos/benchmarks/__init__.py:6:from pykronos import KronosClient
kronos/kronos/core/marshal.py:8:  # json.loads would. We use it in pykronos to speed up event parsing, use it
kronos/README.md:29:Then, from the `chronology/pykronos` directory, run a `python` shell, and
kronos/README.md:36:from pykronos.client import KronosClient
kronos/README.md:53:out the more detailed [pykronos example](../pykronos/).
kronos/requirements.txt:9:pykronos==0.6.0
kronos/scripts/load_test_elections_data.py:9:from pykronos import KronosClient
kronos/scripts/load_test_elections_data.py:10:from pykronos import TIMESTAMP_FIELD
kronos/setup.py:41:                                      'pykronos*',
```

Comment (marcua) [inline kronos/requirements.txt]: it doesn't really look like kronos depends on it then.  that said, i don't see much harm in bringing it in for kind souls that want to test things out.  i'm ok with this change.

**Comment by marcua** [comment]
lgtm

**Comment by derek-schultz** [comment]
Forgot about this. Still okay to merge?

**Comment by usmanm** [comment]
Yup, looks good.

**Comment by marcua** [comment]
+1

**Pull request merged**